Server.Logic/Managers/RecipeManager.cs

```diff
@@ -178,21 +187,22 @@ public async Task<IManagerActionResult<RecipesComment[]>> GetRecipeCommentsPaged
         return new ManagerActionResult<RecipesComment[]>(await _pageingManager.GetPagedInfo(comments, page, pageSize).ToArrayAsync(), ResultEnum.OK);
     }
 
-    public async Task<IManagerActionResult<RecipeOutViewModel[]>> GetRecipesPagedAsync(int page, int pageSize, string searchName, int[] categoryIds)
+    public async Task<IManagerActionResult<RecipeOutViewModel[]>> GetRecipesPagedAsync(int page, int pageSize, string searchName, int[] categoryIds, string user)
     {
-        IQueryable<Recipe> data = _dbInfo.Recipes.Include(x => x.Categories).Include(x => x.Users);
```

Comment:
bez tego .Include(Users) w createBasicViewModel .Users będzie zawsze puste

Reply:
Ale jest bez (to jest własnie usunięte xD)

Reply:
No a powinno być, potem tu
`            NumberOfLikes = recipe.Users.Count,
            IsLiked = recipe.Users.Any(x => x.Login == user),`
Będziesz mieć puste